### PR TITLE
Feature search form scale filter

### PR DESCRIPTION
**Pull request by DubrovinPavel**

### CODE REVIEW

File: addon/templates/components/flexberry-edit-layermap.hbs [hunk 1]
```diff
@@ -72,6 +72,19 @@
     </div>
     <div class="field">
       <label>
+        {{scaleTextboxCaption}}
```
Comment (almostcake) [inline addon/templates/components/flexberry-edit-layermap.hbs]: Is scale required for group layers, @kuzkok ?

File: tests/integration/components/flexberry-dropdown-input-test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  integration: true
+});
+
+test('it renders', function(assert) {
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  this.render(hbs`{{flexberry-dropdown-input}}`);
+
+  assert.equal(this.$().text().trim(), '(нет значения)');
```
Comment (almostcake) [inline tests/integration/components/flexberry-dropdown-input-test.js]: it is better to store placeholder in locale too, and get it there

File: app/templates/new-platform-flexberry-g-i-s-layer-metadata-e.hbs [hunk 3]
```diff
@@ -43,6 +43,7 @@
     layer=(hash
       type=model.type
       name=model.name
+      scale=model.scale
```
Comment (almostcake) [inline app/templates/new-platform-flexberry-g-i-s-layer-metadata-e.hbs]: scale will not save to the model. look at [_mutateModel](https://github.com/Flexberry/ember-flexberry-gis/blob/feature-search-form-scale-filter/addon/controllers/new-platform-flexberry-g-i-s-layer-metadata-e.js#L64) method

File: addon/controllers/gis-search-form.js [hunk 4]
```diff
@@ -172,6 +199,54 @@ export default Ember.Controller.extend({
...
+    },
+
+    /**
+      Handles scale filter keyDown action.
+
+      @method actions.scaleFilterKeyDown
+    */
+    scaleFilterKeyDown(e) {
+      let key = e.which;
+      return (key === 8 || key === 9 || key === 46 || (key >= 37 && key <= 40) ||
```
Comment (almostcake) [inline addon/controllers/gis-search-form.js]: comments

File: addon/templates/components/flexberry-edit-layermap.hbs (hunk 1 above)
Comment (kuzkok) [reply addon/templates/components/flexberry-edit-layermap.hbs]: no

**Pull request merged**